lib/validators/url_validator.rb

```diff
@@ -24,10 +24,41 @@ def validate_each(record, attribute, value)
         unless value =~ regexp
           record.errors[attribute] << options[:message]
```

Comment:
Maybe there should be a return here too?  Or else options[:message] could be pushed a second time later on?